apps/www/_blog/2025-04-01-supabase-edge-functions-deploy-dashboard-deno-2-1.mdx

```diff
@@ -0,0 +1,121 @@
+    light: '/images/blog/launch-week-14/day-2-edge-functions/download-function.png',
+  }}
+/>
+
+<Admonition type="caution">
+
+The Dashboard's Edge Function editor currently does not support versioning or rollbacks. We recommend using it only for quick testing and prototypes. When you’re ready to go to production, store Edge Functions code in a source code repository (e.g., git) and deploy it using one of the [CI integrations](https://supabase.com/docs/guides/functions/cicd-workflow).
+
+</Admonition>
+
+
+## Testing Edge Functions from the Supabase Dashboard
+
+We are introducing a built-in tool for testing your Edge Functions from the Supabase Dashboard. You can execute your Edge Function with different request payloads, headers, and query parameters. The built-in tester returns the response status, headers, and body. 
+
+<Img
+  wide
+  alt="Test Functions from Dashboard"
+  src={{
+    dark: '/images/blog/launch-week-14/day-2-edge-functions/test-function.png',
+    light: '/images/blog/launch-week-14/day-2-edge-functions/test-function.png',
+  }}
+/>
+
+With the built-in editor and tester, you have a streamlined workflow for creating, testing, and refactoring your Edge Functions without leaving the Supabase Dashboard.
+
+## Deploying Edge Functions no longer requires Docker
+
+By popular request, you can now deploy Edge Functions from the Supabase CLI with the `--use-api` flag, which will not use Docker. We will make this the default behavior in future releases (with a `--use-docker` flag as a fallback option.)  
+
+```bash
+supabase functions deploy MY_FUNCTION --use-api
+```
+
+## New APIs for Deploying Edge Functions
+
+The ability to deploy without Docker in both the Edge Functions editor and Supabase CLI are made possible by new APIs we introduced to deploy and update Edge Functions. These APIs are publicly available for you to build custom integrations and workflows.
+
+You can check [the Changelog announcement](https://github.com/orgs/supabase/discussions/33720) for more details and official references to these API endpoints. 
+
+## Deno 2.1 Preview
+
+Last but not least, we have added Deno 2.1 support for Supabase Edge Runtime. With Deno 2.1, you can use built-in Deno commands to scaffold a new project, manage dependencies, run tests, and lints.
+
+Check [our guide on how to start using Deno 2.1](https://github.com/orgs/supabase/discussions/34054) tooling for your Edge Functions.
+
+<Admonition type="info">
+
+Note that the Supabase hosted platform is still using Deno 1.45. In the coming weeks, we will provide more details on deploying Deno 2.1 projects in the hosted platform.
+
+</Admonition>
+
+## Conclusion
+
+These changes to Supabase Edge Functions make it easier and more accessible for all developers to build powerful functionality into their applications.
+
+- Read the [Edge Functions documentation](https://supabase.com/docs/guides/functions) to learn more
+- Watch the Edge Functions changes video
```

Comment:
this the youtube video? we normally embed it towards the top of the blog post.

Reply:
Yeah, we should embed the video in every main stage launch. Here is the video URL:
https://youtu.be/Pn7xyTbsxPI